flash/tabular/classification/input.py

```diff
@@ -65,3 +65,31 @@ def load_data(
             return super().load_data(
                 read_csv(file), categorical_fields, numerical_fields, target_fields, parameters, target_formatter
             )
+
+
+class TabularClassificationListInput(TabularDataFrameInput, ClassificationInputMixin):
```

Comment:
`TabularDataFrameInput` already has the `ClassificationInputMixin` right? So this can be:
```suggestion
class TabularClassificationListInput(TabularDataFrameInput):
```

Reply:
Fixed :)